MAUI/DataGrid/migration.md

```diff
@@ -222,6 +222,11 @@ To make migration from [Xamarin SfDataGrid](https://www.syncfusion.com/xamarin-u
 <td>{{'[NumericButtonSelectionTextColor]()'| markdownify }}</td>
 <td>This enables users to customize the text color of the currently selected numeric button in the view.</td>
 </tr>
+<tr>
+<td>{{'[EnableGridPaging()](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfDataGrid.XForms.DataPager.SfDataPager.html#Syncfusion_SfDataGrid_XForms_DataPager_SfDataPager_EnableGridPaging)'| markdownify }}</td>
+<td>-</td>
+<td>This property is not available in MAUI DataGrid. Even this property is working based on UseOnDemandPaging property. So, we can use that properly directly.</td>
```

Comment:
This property isn't available because the `UseOnDemandLoading` property alone is sufficient to enable on-demand paging in the MAUI DataGrid.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/c22c222a-e753-4553-835c-1f44ebf73ef5)
Had updated the content.